src/plays/react-todo-app/ReactTodoApp.jsx

```diff
@@ -0,0 +1,84 @@
+import { getPlayById } from 'meta/play-meta-util';
+
+import PlayHeader from 'common/playlists/PlayHeader';
+
+import { useState, useEffect } from 'react';
+
+// css file
+import './react-todo.css';
+
+// components
+import Input from './components/Input';
+import TodoList from './components/TodoList';
+
+function ReactTodoApp(props) {
+  // Do not remove the below lines. 
+  // The following code is to fetch the current play from the URL
+  const { id } = props;
+  const play = getPlayById(id);
+
+  // Your Code Start below.
+
+  const [input, setInput] = useState("");
+  const [todos, setTodos] = useState([]);
+  const [status, setStatus] = useState("all");
+  const [filtered, setFiltered] = useState([]);
+
+  useEffect(() => {
+
+    const getTodos = () => {
+      if (localStorage.getItem("todos") === null) {
+        localStorage.setItem("todos", JSON.stringify([]));
+      }
+      else {
+        let localTodo = JSON.parse(localStorage.getItem("todos"));
+        setTodos(localTodo);
+      }
+    }
+
+    getTodos();
+  }, [])
+
+  useEffect(() => {
+
+    const filterHandler = () => {
+      switch (status) {
+        case "completed":
+          setFiltered(todos.filter(todo => todo.completed === true));
+          break;
+        case "uncompleted":
+          setFiltered(todos.filter(todo => todo.completed === false));
+          break;
+        default:
+          setFiltered(todos);
+          break;
+      }
+    }
+
+    const saveLocalTodos = () => {
+      localStorage.setItem("todos", JSON.stringify(todos));
```

Comment:
Change the key name by prefixing your github id. Soemone else may add a todo in future with will override it.